supervisory-authorities/bgcpdp.json

```diff
@@ -3,7 +3,7 @@
     "relevant-countries": [
         "bg"
     ],
-    "name": "Commission for Personal Data Protection Bulgaria",
+    "name": "Commission for Personal Data Protection Bulgaria ",
```

Comment:
🚫 **[recordlint]** <[trim-string-fields](https://github.com/datenanfragen/data#data-formats)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Value of string field `name` is not trimmed (it contains leading or trailing whitespace).
```suggestion
    "name": "Commission for Personal Data Protection Bulgaria",
```